verification/curator-service/ui/src/components/new-case-form-fields/Location.tsx

```diff
@@ -66,6 +67,16 @@ class Profile extends React.Component<Props, {}> {
                             : 'N/A'}
                     </p>
                 </div>
+                <div className={classes.column}>
+                    <p>
+                        <Typography variant="caption">Admin area 3</Typography>
+                    </p>
+                    <p>
+                        {this.props.location?.adminArea3
```

Comment:
This can be `{this.props.location?.adminArea3 ?? 'N/A'}` and elsewhere - missed that in the last review!

Reply:
nice! done.